system/interceptors/SES.cfc

```diff
@@ -645,6 +647,17 @@ Description :
 		<cfreturn instance.enabled/>
 	</cffunction>
 
+    <!--- Getter/Setter PathsForTesting --->
+    <cffunction name="setIntegrationTestingPaths" access="public" output="false" returntype="any" hint="Set whether the interceptor is enabled or not and return itself">
```

Comment:
@elpete I know this will do the job, but my instinct tells me that we should not place any direct references to integration testing on production MVC code.  So now that you showed that this code works, maybe we can refactor it in such a way, where the `BaseTestCase` can manipulate the interceptor's path locations instead of adding a concrete integration point.